### DISCUSSION TITLE
"Delta with previous objective"

### DISCUSSION THREAD

**Discussion by FurtadoPires** [ReactiveDrop/reactivedrop_translations#1511] [Q&A]
From `resource\reactivedrop_.txt`
Line 610 / 611,  `"Objective %s1 complete! \nTime: %s2 Delta with previous objective: %s3\n"`
that does the phrase "Delta with previous objective" means exactly?

**Comment by jhh8** [comment]
it means how much time has passed since previous objective completion, to the one that just got completed. time between objective completions

**Reply by FurtadoPires** [reply]
Is "Delta with" an expression, or it's a typo? Personally never heard of it.

**Reply by Duke-M-commits** [reply]
Now that you're mentioning it, shouldn't it be "Delta to previous objective"?
I mean, we're not giving the order of calculating the delta with the value, we're telling the result of the calculation to the user.

**Reply by FurtadoPires** [reply]
> shouldn't it be "Delta to previous objective"?

The original string says "Delta with previous objective" but I don't know what it means and I can't find a reference of this being used elsewhere.

Is this some military gibberish? And if it's the case, for which term could I replace to keep the original meaning?

**Reply by Duke-M-commits** [reply]
It's a term used in mathematics, describing _a (rate of) change_ between 2 things.
https://sciencing.com/calculate-delta-between-two-numbers-5893964.html

Additionally, "original string" doesn't have to mean it's right, there are sometimes mistakes in the old stuff too, when unsure about something, just question it, can't get worse :D
